LearningMissionSimulation/PlaygroundGarush.cs

```diff
@@ -5,8 +5,78 @@
 
 namespace LearningMissionSimulation
 {
-    public class PlaygroundGarush
+    public class PlaygroundGarush : ISimulation
     {
-    
+        List<Account> pendingAccountList = new List<Account>();
+        List<Student> studentList = new List<Student>();
+        List<Instructor> instructorList = new List<Instructor>();
+
+        Dictionary<Guid, Account> accountDictionary = new Dictionary<Guid, Account>();
+        public void CreateAccounts(int accountCount)
+        {
+            for (int i = 0; i < accountCount; i++)
+            {
+                Account account = ObjectGenerator.GenerateAccount();
+
+                accountDictionary.Add(account.Id, account);
+                if (account.Role == Role.Student)
+                {
+                    Student student = ObjectGenerator.GenerateStudent(account.Id);
+                    studentList.Add(student);
+                    if (student.Gender == Gender.Female && account.Status == Status.Pending)
+                    {
+                        pendingAccountList.Add(account);
+                    }
+                }
+                else if (account.Role == Role.Instructor)
+                {
+                    Instructor instructor = (ObjectGenerator.GenerateInstructor(account.Id));
+                    instructorList.Add(instructor);
+                }
+                if ((account.Role == Role.Student || account.Role == Role.Instructor) && account.Status == Status.Pending)
```

Comment:
This if block is absolutely redundant:
 if ((account.Role == Role.Student || account.Role == Role.Instructor) && account.Status == Status.Pending)
                {
                    pendingAccountList.Add(account);

                }
Please remove it.